examples/llm_compression/torch/qat_with_lora/NLSDownstreamTasks.md

```diff
@@ -0,0 +1,94 @@
+# NLS Tuning with Downstream Tasks
+
+<p align="center">
+  <img src="/examples/llm_compression/torch/qat_with_lora/pics/lora_vs_nls.png" alt="LoRA vs NLS" width="400"/>
+</p>
+
+[main_nls.py](./main_nls.py) script supports fine-tuning and evaluating a language model with quantization-aware training and **Neural Low-Rank Adapter Search (NLS)** proposed by [Shears](https://arxiv.org/abs/2404.10934) and [SQFT](https://arxiv.org/abs/2410.03750) on various downstream tasks. For example, to run the script for the task [openbookqa](https://huggingface.co/datasets/allenai/openbookqa), you can use the following command:
```

Comment:
Could you provide some results by training speed? comparing with LoRA mo NLS?

Reply:
@alexsu52 @ljaljushkin Yes, there is minimal overhead during training. For instance, fine-tuning the compressed Llama-3.2-3B-Instruct model for 3 epochs on arc-challenge takes 161.83 seconds with LoRA and 164.89 seconds with NLS. Thanks!